applications/FluidTransportApplication/python_scripts/coupled_fluid_transport_solver.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+
+# Importing the base class
+from KratosMultiphysics.ConvectionDiffusionApplication.coupled_fluid_thermal_solver import CoupledFluidThermalSolver
+from KratosMultiphysics.python_solver import PythonSolver
+
+from importlib import import_module
+
+def CreateSolver(main_model_part, custom_settings):
+    return CoupledFluidTransportSolver(main_model_part, custom_settings)
+
+class CoupledFluidTransportSolver(CoupledFluidThermalSolver):
+
+    def __init__(self, model, custom_settings):
+
+        self._validate_settings_in_baseclass=True # To be removed eventually
+
+        PythonSolver.__init__(self, model, custom_settings)
+
+        ## Get domain size
+        self.domain_size = self.settings["domain_size"].GetInt()
+
+        from KratosMultiphysics.FluidDynamicsApplication import python_solvers_wrapper_fluid
```

Comment:
Please move all the imports to the top of the file. Aside of being a good practise according to PEP-8 style guide, we had some recursive imports issues in the past (that shouldn't happen now)  because of this.